alpha/apps/kaltura/lib/live/kBroadcastUrlManager.php

```diff
@@ -271,10 +274,19 @@ public function getEncryptedSrtPass(LiveStreamEntry $entry)
 		return kDeliveryUtils::urlsafeB64Encode($encryptedToken);
 	}
 
+	private static function addLiveBackendId($id)
```

Comment:
code convention is to use protected and not private

Reply:
changed